src/jimgw/single_event/detector.py

```
@@ -9,9 +9,12 @@
 from beartype import beartype as typechecker
 from scipy.interpolate import interp1d
 from scipy.signal.windows import tukey
+from . import data as jd
+from typing import Optional
 
 from jimgw.constants import C_SI, EARTH_SEMI_MAJOR_AXIS, EARTH_SEMI_MINOR_AXIS
 from jimgw.single_event.wave import Polarization
+import logging
```

Ruff (0.8.2)

`logging` imported but unused

Remove unused import: `logging`

(F401)